### ISSUE TITLE
Form Builder settings window, scroll option missing in certain situations

### ISSUE THREAD

**Issue by mawtex** [Orckestra/C1-CMS-Foundation#565]
When editing the settings of a form (Composite.Forms.FormBuilder.Form) the tabs may have content below the fold, but no scroll bar to show this.

Repro for the below example is adding more action options by editing ~/App_Data/Composite/Configuration/Composite.Forms.FormBuilder.xml and opening up the form editor in a small enough browser window to not have vertical space for all options.

Expected: You can scroll down.
Actual: No way to scroll down, can't choose hidden options. 

Also notice the "perspective scroll button" is visible through this dialog in the lower left corner.
![all visible](https://user-images.githubusercontent.com/2146669/40369048-6b64f0ba-5ddd-11e8-9182-eeec35fc6895.png)
![only top visible](https://user-images.githubusercontent.com/2146669/40369049-6b868b62-5ddd-11e8-8354-3a3386269e59.png)

**Comment by wwwysocki** [comment]
Still partly active. Retested on 20180621.1 (dev)

**Prepare**:

1. Edit `~/App_Data/Composite/Configuration/Composite.Forms.FormBuilder.xml`
2. Below `<Actions/>` add a child element with another [dummy] action (see the markup below for an illustration)
3.  Save the file

```
    <Action name="SomeOtherAction" label="Do Some Other Action" help="Do Some Other Action">
      <f:function name="Composite.Forms.FormBuilder.Handlers.SaveToFile">
      </f:function>
    </Action>
```

**Repro**:

1. Edit a page
2. Insert Function > Composite > Forms > FormBuilder > Form
3. Switch to the Actions tab
4. Resize the browser window (vertically)

**Expected**: The scroll bar appears
**Actual**: 
1. When you click the "Normalize" button on the browser window, everything goes blank
2. When you go on to resize the window vertically, the action blocks reappear but there is no scroll bar

**Note**:

The scroll bar **does** appear if you first resize the window and then do Repro Steps 2-3.

**Comment by wwwysocki** [comment]
The new scenario described in the comment above is logged as a separate issue https://github.com/Orckestra/C1-CMS-Foundation/issues/578

This bug is closed as fixed in "dev" --> Verified on  20180621.1 (dev)
